_content/canon/naughty-jataka_dhammika.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "A Naughty Jataka"
```

Comment:
As always with canonical works, filename is SC ID and title is "ID Pāḷi: English"

In this case, `ja273+cmy.md` and `Ja 273 Kacchapa Jātaka: The Tortoise (A Naughty Jātaka)`